_i18n/template/resources/developer-guides/wallet-rpc.md

```diff
@@ -78,34 +87,56 @@ Note: "atomic units" refer to the smallest fraction of 1 XMR according to the mo
 
 Return the wallet's balance.
 
-Inputs: *None*.
+Inputs:
+
+* *account_index* - unsigned int; Return balance for this account.
 
 Outputs:
 
 * *balance* - unsigned int; The total balance of the current monero-wallet-rpc in session.
 * *unlocked_balance* - unsigned int; Unlocked funds are those funds that are sufficiently deep enough in the Monero blockchain to be considered safe to spend.
+* *multisig_import_needed* - boolean; True if balance transferred to a multisig wallet.
```

Comment:
Incorrect. It's true if importing multisig data from other signers is needed to get correct balance

Reply:
Thanks for catching this, will update with the correct information.